### ISSUE TITLE
Uncaught SyntaxError: Unexpected token 'export'

### ISSUE THREAD

**Issue by nstepien** [PepsRyuu/nollup#141]
I get this error coming from this eval:
![image](https://user-images.githubusercontent.com/567105/91859893-e15add00-ec62-11ea-8956-5167c0aa29ee.png)

Such modules with empty exports come from TypeScript files exporting only types, ts will remove all the types, leaving only an empty export line, and nollup doesn't seem to handle it well.

For now I'm patching out these empty exports manually with a custom rollup plugin, so nollup can keep working:
```js
    (process.env.NODE_ENV !== 'production') && {
      name: 'patch-empty-exports',
      transform(code, id) {
        if (/\.[jt]sx?$/.test(id)) {
          const newCode = code.replace(/export\s+\{\s*\}/, ';');
          if (newCode !== code) {
            return { code: newCode };
          }
        }
      }
    },
```

**Comment by PepsRyuu** [comment]
Nice catch! Should be easy enough to resolve. Will get to it as soon as I can!

**Comment by PepsRyuu** [comment]
https://github.com/PepsRyuu/nollup/pull/143

If you could give this a quick try and let me know if it solves your problem I'd greatly appreciate it! :)

**Comment by nstepien** [comment]
Works great!

**Comment by PepsRyuu** [comment]
Released in ```0.13.10```
